src/base/matrix.rs

```diff
@@ -161,16 +157,18 @@ pub type MatrixCross<T, R1, C1, R2, C2> =
 #[cfg_attr(
     feature = "rkyv-serialize-no-std",
     derive(Archive, rkyv::Serialize, rkyv::Deserialize),
```

Comment:
Do we want to derive `rkyv::Portable` when possible ? I'm not sure what the implications are :/